flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_PRESS.js

```diff
@@ -203,17 +203,17 @@ var A320_Neo_LowerECAM_PRESS;
             const inletValvePosition = SimVar.GetSimVarValue("L:VENT_INLET_VALVE", "Percent");
             const outletValvePosition = SimVar.GetSimVarValue("L:VENT_OUTLET_VALVE", "Percent");
             const safetyValvePosition = (SimVar.GetSimVarValue("L:SAFETY_VALVE_1", "Bool") || SimVar.GetSimVarValue("L:SAFETY_VALVE_2", "Bool"));
-            const activeSystem = (SimVar.GetSimVarValue("L:CPC_SYS1", "Bool")) ? 1 : 2;
+            const activeSystem = SimVar.GetSimVarValue("L:A32NX_ACTIVE_CPC_SYS", "Number");
 
-            const cabinVSValue = fastToFixed(SimVar.GetSimVarValue("L:CABIN_ALTITUDE_RATE", "Feet per second"), 0);
-            const pressureDelta = SimVar.GetSimVarValue("L:DELTA_PRESSURE", "PSI");
-            const cabinAltitude = fastToFixed(SimVar.GetSimVarValue("L:CABIN_ALTITUDE", "Feet"), 0);
+            const cabinVSValue = fastToFixed(SimVar.GetSimVarValue("L:A32NX_CABIN_VS", "Feet per minute"), 0);
+            const pressureDelta = SimVar.GetSimVarValue("L:A32NX_CABIN_DELTA_PRESSURE", "PSI");
+            const cabinAltitude = fastToFixed(SimVar.GetSimVarValue("L:A32NX_CABIN_ALTITUDE", "feet"), 0);
             const pressureDeltaDecimalSplit = pressureDelta.toFixed(1).split(".", 2);
-            const outletValveOpenPercent = SimVar.GetSimVarValue("L:OUTFLOW_VAVLE_PCT", "Percent");//it is defined as "VAVLE" in the wasm, not a typo on my part
+            const outletValveOpenPercent = SimVar.GetSimVarValue("L:A32NX_OUTFLOW_VALVE_OPEN_PERCENTAGE", "Percent");
 
             const leftPackState = (!SimVar.GetSimVarValue("L:A32NX_AIRCOND_PACK1_TOGGLE", "bool") && SimVar.GetSimVarValue("ENG COMBUSTION:1", "Bool"));
             const rightPackState = (!SimVar.GetSimVarValue("L:A32NX_AIRCOND_PACK2_TOGGLE", "bool") && SimVar.GetSimVarValue("ENG COMBUSTION:2", "Bool"));
-            const landingElev = SimVar.GetSimVarValue("L:A32NX_LANDING_ELEVATION", "feet");
+            const landingElev = (SimVar.GetSimVarValue("L:A32NX_LANDING_ELEVATION", "feet") !== -2000) ? SimVar.GetSimVarValue("L:A32NX_LANDING_ELEVATION", "feet") : SimVar.GetSimVarValue("L:A32NX_AUTO_LANDING_ELEVATION", "feet");
```

Comment:
I don't think this would be an issue as the `-2000` comes directly from the overhead panel "initial position" (represented by the `L:A32NX_LANDING_ELEVATION` Simvar). The idea here is to have the landing elevation in auto as long as the overhead knob hasn't been moved, if you move the knob the landing elevation changes to manual. This is how the real airplane behaves, but the code will probably need to be improved in the future to make it clearer and more robust.

Reply:
I see. This isn't then represented as say `1999.999999999999999999999999965`? If it works it work? 😀